tests/unit/test_update_checker.py

```
@@ -95,6 +95,18 @@ def test_get_release_commit(self, config:Config) -> None:
         with patch("requests.get", return_value = MagicMock(json = lambda: {"target_commitish": "e7a3d46"})):
             assert checker._get_release_commit("latest") == "e7a3d46"
 
+    def test_request_timeout_uses_config(self, config:Config, mocker:"MockerFixture") -> None:
```

Ruff (0.14.4)

Remove quotes from type annotation

Remove quotes

(UP037)